### PR TITLE
Face Detection Library Replacement

### PR DESCRIPTION
**Pull request by oguzhanbsolak**
Changelog:
    - Hawk-eyes replaced with face-detection package
    - Requirements.txt updated according to package change

### CODE REVIEW

File: datasets/vggface2.py [hunk 1]
```diff
@@ -23,9 +23,8 @@
 from torchvision import transforms
 
 import cv2
+import face_detection
 import kornia.geometry.transform as GT
```
Comment (ermanok) [inline datasets/vggface2.py:27]: I think the copyright text should be updated for 2024.

**Review by ermanok — APPROVED** [review]
Looks OK after the fix for copyright text,

File: datasets/vggface2.py (hunk 1 above)
Comment (oguzhanbsolak) [reply datasets/vggface2.py:27]: Realized in the last commit.

**Comment by oguzhanbsolak** [comment]
Added faceID backbone checkpoint-related improvements with the last commit.

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
In models/model_irse_drl.py, please consolidate the common code.

**Comment by oguzhanbsolak** [comment]
> In models/model_irse_drl.py, please consolidate the common code.

Realized with the last commit.

**Pull request merged**